apps/docs/content/guides/platform/compute-add-ons.mdx

```diff
@@ -63,19 +63,19 @@ Performance Limits:
 
 Disk throughput is capped at 19,000 Mbps and IOPS is capped at 80,000 IOPS for all compute add-ons.
 
-| Compute add-on | Max Disk Throughput | Baseline Disk Throughput | Max IOPS    | Baseline IOPS |
-| -------------- | ------------------- | ------------------------ | ----------- | ------------- |
-| Nano (free)    | 2,085 Mbps          | 43 Mbps                  | 11,800 IOPS | 250 IOPS      |
-| Micro          | 2,085 Mbps          | 87 Mbps                  | 11,800 IOPS | 500 IOPS      |
-| Small          | 2,085 Mbps          | 174 Mbps                 | 11,800 IOPS | 1,000 IOPS    |
-| Medium         | 2,085 Mbps          | 347 Mbps                 | 11,800 IOPS | 2,000 IOPS    |
-| Large          | 4,750 Mbps          | 630 Mbps                 | 20,000 IOPS | 3,600 IOPS    |
-| XL             | 4,750 Mbps          | 1,188 Mbps               | 20,000 IOPS | 6,000 IOPS    |
-| 2XL            | 4,750 Mbps          | 2,375 Mbps               | 20,000 IOPS | 12,000 IOPS   |
-| 4XL            | 4,750 Mbps          | 4,750 Mbps               | 20,000 IOPS | 20,000 IOPS   |
-| 8XL            | 9,500 Mbps          | 9,500 Mbps               | 40,000 IOPS | 40,000 IOPS   |
-| 12XL           | 14,250 Mbps         | 14,250 Mbps              | 50,000 IOPS | 50,000 IOPS   |
-| 16XL           | 19,000 Mbps         | 19,000 Mbps              | 80,000 IOPS | 80,000 IOPS   |
+| Compute size | Max Disk Throughput | Baseline Disk Throughput | Max IOPS    | Baseline IOPS |
```

Comment:
This one's something that I asked to be standardized. We use these very inconsistently. e.g. this page is titled Compute Add-ons. I'm fine with either one, just that it be used consistently.

Reply:
Agree, will do that separately. Nano and Micro are no compute add-ons, so compute size is more fitting here, will streamline separately